README.md

```diff
@@ -35,6 +35,8 @@ var raf = require('raf')
 
 Shorthand to polyfill `window.requestAnimationFrame` and `window.cancelAnimationFrame` if necessary (Polyfills `global` in node).
 
+Alternatively you can require `raf/polyfill`.
```

Comment:
Could you expand on this in the docs — noting that `require('raf/polyfill')` is equivalent to `require('raf').polyfill()` (i.e., that the new API immediately polyfills, & exports nothing)?